### ISSUE TITLE
Can one connect them to a Windows pc after hacking them with ATC firmware? #438

### ISSUE THREAD

**Issue by orenatan** [JsBergbau/MiTemperature2#95]
Can one connect them to a Windows pc after hacking them with ATC firmware?

**Comment by JsBergbau** [comment]
Yes this is possible. You can use the Telink Flasher https://atc1441.github.io/TelinkFlasher.html with Chrome in Windows as well. You have to enable experimental features, see https://reelyactive.github.io/pareto-anywhere/enable-web-bluetooth-scanning/
